### ISSUE TITLE
Office templates list endpoints

### ISSUE THREAD

**Issue by basilelegal** [tracim/tracim#2075]
Create a new endpoint to list templates available:

`/workspace/<id>/wopi/files/templates` => 
```
{
  'available_types': ['calc', 'text', 'pres'],
  'available_templates': {
    'calc': ['default.ods'],
    'text': ['default.odt'],
    'pres': ['default.odp']
  }
}

**Comment by inkhey** [comment]
@basilelegal Where template should be stored, is it wopi protocol related
can you explain interaction between frontend, backend and collabora here ?

**Comment by inkhey** [comment]
After discussion:
- This list is not wopi related
- This should not be either collabora related as template feature may be useful for other case too.

Current implementation is:

`/api/v2/files/templates`

return :
```
{
  "categories": [
    "calc"
  ],
  "file_templates": [
    {
      "category": "calc",
      "template_name": "default.odt"
    }
  ]
}

```

**Comment by inkhey** [comment]
After discussion, idea is to have different feature about template, one for template for online_edition, one other for any file, we discuss here about online_edition specific case.

future endpoint should return only list of file of the directory of default template for online-edition, all intelligence is in frontend here.

Future endpoint draft :
`
GET /api/v2/online-edition/templates  
`
```
{
  'templates_name': [ 'write.odt', 'spreedsheat.ods']
}

```

**Comment by inkhey** [comment]
New current implementation in https://github.com/tracim/tracim/tree/poc/collabora_integration_v2 is:

Endpoint 
`/api/v2/collaborative-document-edition/templates`
```
{
  "file_templates": [
    "string"
  ]
}
```

**Comment by inkhey** [comment]
this is fixed by #2139

**Comment by PhilippeAccorsi** [comment]
Verified.
